go.mod

```diff
@@ -3,16 +3,18 @@ module github.com/newrelic/infrastructure-agent
 go 1.19
 
 require (
-	github.com/Microsoft/go-winio v0.5.1
+	github.com/Microsoft/go-winio v0.5.2
 	github.com/StackExchange/wmi v0.0.0-20180116203802-5d049714c4a6
 	github.com/antihax/optional v1.0.0
 	github.com/aws/aws-sdk-go v1.44.69
 	github.com/beevik/ntp v0.3.0
+	github.com/containerd/containerd v1.5.10
 	github.com/coreos/go-systemd/v22 v22.3.2
-	github.com/docker/docker v23.0.1+incompatible
+	github.com/docker/docker v1.4.2-0.20190924003213-a8608b5b67c7
```

Comment:
Any reason why this module was downgraded? We recently bump it to the latest version to fix some vulnerabilities, I am afraid they will come back.